service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderController.java

```diff
@@ -116,7 +143,7 @@ public AboutToStartOrSubmitCallbackResponse handleMidEventIssueDate(@RequestBody
         return respond(caseDetails);
     }
 
-    @PostMapping("/mid-event")
+    @PostMapping({"/mid-event", "/service-route/mid-event"})
```

Comment:
@KiranVarma02 see that, nice little trick, could be something you can use?

Reply:
yes, nice trick 👍 . In my case, 
In old endpoint - I am validating only PBA
In new endpoint - I am validating both PBA and date. 

Looks like it doesn't work in my case!